purchase_blanket_order/models/blanket_orders.py

```diff
@@ -298,7 +299,7 @@ def action_cancel(self):
                                 "Try to cancel them before."
                             )
                         )
-            order.write({"state": "expired"})
+            order.write({"state": "expired", "cancelled": True, "confirmed": False})
```

Comment:
Dies not make sense to write the state 'expired' because the "cancelled" and "confirmed" fields will drive the correct state during the computation

Reply:
@JordiBForgeFlow It's done now